modules/apps/bean-portlet/bean-portlet-cdi-extension-sample-test/src/main/webapp/WEB-INF/liferay-plugin-package.properties

```diff
@@ -0,0 +1,9 @@
+author=Liferay, Inc.
+Bundle-Version: 1.0.0
+change-log=
+licenses=LGPL
+module-group-id=liferay
+module-incremental-version=1
+name=Sample Bean Portlet
+page-url=http://www.liferay.com
+tags=sample
```

Comment:
@Kvale1733, refer to `PluginPackageUtil#_readPluginPackageServletContext()`. The `Bundle-Version: 1.0.0` is invalid property if `liferay-plugin-package.properties` existed, please remove it. And then add the below properties:
```
module-version=1.0.0
```
And change `module-group-id=liferay` for `module-group-id=com.liferay`